### DISCUSSION TITLE
iOS app

### DISCUSSION THREAD

**Discussion by HippoBloke** [photoview/photoview#1106] [General]
Any reason why the iOS application is unavailable in the UK ?

Thanks
Hippo

**Comment by kkovaletp** [comment]
As far as I know, it is not actively maintained. There is a dedicated repo for it: https://github.com/photoview/photoview-ios - feel free to submit (or comment if already exists) an issue there

**Comment by joe2470** [comment]
Same situation in Germany.
Unfortunately in there is no option in app rep to open a discussion. So would be nice if somebody could provide here an update. 
Thank you!

**Reply by kkovaletp** [reply]
There are no updates, the status is still the same. You can check it by yourself: navigate to the app repo https://github.com/photoview/photoview-ios and check for the activity and the latest commit in the `master`.

**Reply by joe2470** [reply]
Thank your for your answer.
But unfortunately I still don't get how to install the app on an iPhone?

**Reply by kkovaletp** [reply]
I don't know. I don't have an iPhone and never built apps for it.

**Comment by jparris** [comment]
Just commenting to say that the App is no longer available in the US.

**Reply by kkovaletp** [reply]
The mobile app project has not been actively maintained for years, so it is expected...
